src/pages/ProductListPage/FilterSection/FilterProduct.jsx

```diff
@@ -1,82 +1,263 @@
+                    console.log(object.categoryType.includes(item.categoryType));
+                    return ((  object.categoryType.includes(item.categoryType)) 
+                    || (object.feature.includes(item.feature))
+                    || (object.rating  ? item.rating >= object.rating : false)
+                    || (object.productType  ? item.productType === object.productType : false)
+                    || (object.productStockType  ? item.stockType ===object.productStockType : false) )
+                }))
+            // })
+            setProductList(
+                [...products.filter((item) =>{
+                    console.log(object.categoryType.includes(item.categoryType));
+                    return (object.categoryType.includes(item.categoryType)
+                    || (object.feature.includes(item.feature))
+                    || (object.rating  ? item.rating >= object.rating : false)
+                    || (object.productType  ? item.productType === object.productType : false)
+                    || (object.productStockType  ? item.stockType ===object.productStockType : false) )
+            
+            })])
+        })()
+            
+    }
+
+    function SortByPrice(orderBool){
+        setProductList(
+            [...ProductList.sort((a,b)=>{
+                if(orderBool){
+                    return a.discountedPrice < b.discountedPrice ? -1 : a.discountedPrice > b.discountedPrice ? 1 :0;
+                }
+                return a.discountedPrice > b.discountedPrice ? -1 : a.discountedPrice < b.discountedPrice ? 1 :0;
+            })]
+        )
+    }
+    return (
+        <aside className="aside">
+            <div className="aside-header">
+                <h3>Filters</h3>
+                <a>Clear</a>
             </div>
-            <div className="rating-container">
-                <h4>Rating</h4>
-                <div>
-                    <input type="radio" id="4stars" name="drone" value="4stars"
-                        checked/>
-                    <label htmlFor="4stars">4 stars and above</label>
+            <div className='aside-body'>
+                {/* <div className="price-container">
+                    <h4>Price</h4>
+                    <input type="range" value="50" max="100" min="0" className="price-slider"/>
+                </div> */}
+                {/*  categoryType:"Football Cricket Gym Running Regular */}
+                <div className="product-page-category">
+                    <h4>Category</h4>
+                    <div>
+                        <input className="checkbox-sneakers" onClick={()=>{
+                            setCategoryType(()=>( categoryType.includes("Football") 
+                                                    ? [...categoryType.filter(item=>item !=="Football")]
+                                                    : [...categoryType,"football"])
+                                                    )}} 
```

Comment:
you can write a utility function here too.

Reply:
okay done